changelogs/unreleased/gh-8069-luajit-fixes.md

```diff
@@ -0,0 +1,7 @@
+## bugfix/luajit
+
+Backported patches from vanilla LuaJIT trunk (gh-8069). In the scope of this
+activity, the following issues have been resolved:
+
+* Fixed loop realigment for dual-number mode
```

Comment:
A dot at the end is missing :)

Reply:
Well, considering the urgency I'll leave this on doc team. Anyway, thanks for noticing!